### ISSUE TITLE
<Space><Space> was identified as  “<Space> 【“

### ISSUE THREAD

**Issue by Tback1** [brookhong/Surfingkeys#1591]
in the 1.0 version add the api. before the api.it's not work as expect

```
api.mapkey('<Space><Space>', 'Choose a tab with omnibar', function() {
    Front.openOmnibar({type: "Tabs"});
});
```

but the key is not two space
![image](https://user-images.githubusercontent.com/8872546/147720773-5c27811b-8ee4-449d-b2c2-2b54fbd406a9.png)

the full setting is in this issus
https://github.com/brookhong/Surfingkeys/issues/1593

**Comment by brookhong** [comment]
Fixed in 1.0.2, thanks for reporting this.

**Comment by Tback1** [comment]
Good Job.Thanks!
